assignment2.txt

```diff
@@ -0,0 +1,17 @@
+1.   a) 5 is equal or less than 4 - false
+	5 does not equal 9 - true
+	5 is larger than 10 - false
+	false || false == FALSE
+
+     b) 5 is larger than 0 - true
+	5 is smaller than 10 - true
+	5 does not equal 6 - true
+	true && true && true == TRUE
+
+     c) 5 equals 1 - false
+	5 is larger than 0 - true
+	false || true == TRUE
+
+2. Yes, because strings can be stored in a variable and that data type can be used in a case.
```

Comment:
Not quite! Strings are an array of characters so to make the comparison, you'd have to iterate through each character (or use strcmp), so you can't use strings. You can use chars though!